src/mousetracker.js

```diff
@@ -2880,11 +2926,10 @@
             }
 
             // A primary mouse button may have been released while the non-primary button was down
-            if (pointsList.contacts > 0 && pointsList.type === 'mouse') {
-                // Stop tracking the mouse; see https://github.com/openseadragon/openseadragon/pull/1223
-                pointsList.contacts--;
-                return true;
```

Comment:
No, because that was just to make sure releasePointer would be called, and that is done through abortContacts now.